src/wasm-linker.cpp

```diff
@@ -381,12 +381,14 @@ void Linker::makeDynCallThunks() {
   }
 }
 
-Function* Linker::generateImportThunk(Name name, const FunctionType* funcType) {
+Function* Linker::getImportThunk(Name name, const FunctionType* funcType) {
+  Name thunkName = std::string("__importThunk_") + name.c_str();
```

Comment:
Oops, we raced and I didn't see the comment before I merged, sorry.
Names with double underscores are reserved for use by the language implementation. So the only conflicts we need to worry about are those that we introduce, via the C/C++ headers, compiler, linker, etc. So I don't think this is a huge issue.
Having said that, it's not a bad idea to check for collisions. I'd rather not start depending on optimizations for correctness.